collectors/python.d.plugin/nvidia_smi/README.md

```diff
@@ -39,6 +39,17 @@ It produces the following charts:
 
 ## Configuration
 
+Turn on the plugin by editing configuration file python.d.conf
```

Comment:
I think we could use same text as in https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/samba/README.md#enable-the-collector